packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -939,6 +968,16 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		}
 	}
 
+	/**
+	 * Get the row index, based on its position in dataSource.data
+	 * @param row - Row to get index
+	 */
+	public getRowIndex(row: any): number | void {
```

Comment:
I think we should return `undefined` instead of `void` so it is clear that no index was found. Normally `void` is used when the flow of the program/function does not return anything... but in this case we should always return something...